cmd/amplifier/main.go

```diff
@@ -14,6 +14,7 @@ const (
 	elasticsearchDefaultURL = "http://localhost:9200"
 	defaultClientID         = ""
 	defaultClientSecret     = ""
+	kafkaDefaultURL         = "kafka:9092"
```

Comment:
Needs to be "localhost:9092" for now (in keeping with our other default hosts). Otherwise it breaks us for local development/testing.